cocos/spine/lib/spine-core.d.ts

```diff
@@ -544,7 +544,7 @@ declare namespace spine {
         isLoadingComplete(clientId: string): boolean;
         dispose(): void;
         hasErrors(): boolean;
-        getErrors(): Map<string>;
+        getErrors(): Map<string, string>;
```

Comment:
different with https://github.com/EsotericSoftware/spine-runtimes/blob/3.8/spine-ts/build/spine-core.d.ts

Reply:
The definition here is not wrong, but when cocos outputs dts, it does not add the namespace, which causes the map of spine to pollute other code.